conda-recipe/run_test.sh

```diff
@@ -15,55 +15,51 @@
 # limitations under the License.
 #===============================================================================
 
-daal4py_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+sklex_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 count=3
 while [[ count -ne 0 ]]; do
-    if [[ -d $daal4py_dir/daal4py/ && -d $daal4py_dir/tests/ && -d $daal4py_dir/examples/daal4py ]]; then
+    if [[ -d $sklex_root/.ci/ && -d $sklex_root/examples/ && -d $sklex_root/tests/ ]]; then
         break
     fi
-    daal4py_dir="$( dirname "${daal4py_dir}" )"
+    sklex_root="$( dirname "${sklex_root}" )"
     count=$(($count - 1))
 done
 
-echo "daal4py_dir=$daal4py_dir"
 if [[ count -eq 0 ]]; then
-    echo "run_test.sh must be in daal4py repository"
+    echo "run_test.sh did not find the required testing directories"
     exit 1
 fi
 
-echo "Start testing ..."
 return_code=0
 
-python -c "import daal4py"
-return_code=$(($return_code + $?))
+if [ -z "${PYTHON}" ]; then
+    export PYTHON=python
```

Comment:
Is this one used anywhere when running on linux? I understood it was meant for the MPI invocation on windows only.

Reply:
For `${PYTHON} -c "from sklearnex import patch_sklearn; patch_sklearn()"` at least. It might be required for some additional tests later, so no need to remove it.

Reply:
Is `${PYTHON}` ever set to something other than "python" when executing the .sh script? If not, would be nice to add it as a comment in the file.

Reply:
It's used by `conda-build`: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process